lib/ansible/modules/cloud/amazon/aws_waf_rule.py

```diff
@@ -295,6 +336,8 @@ def main():
         dict(
             name=dict(required=True),
             metric_name=dict(),
+            type=dict(default='regular'),
```

Comment:
If there are only two valid options you can do:

```suggestion
            type=dict(default='regular', choices=['rate_based', 'regular']),
```

Reply:
Thanks, I will fix that.